src/pkg/zoci/pull.go

```diff
@@ -52,11 +51,14 @@ func (r *Remote) PullPackage(ctx context.Context, destinationDir string, concurr
 	}
 	layersToPull = append(layersToPull, manifest.Config)
 
+	layerSize := oci.SumDescsSize(layersToPull)
+	// TODO (@austinabro321) change this and other r.Log() calls to the proper slog format
+	r.Log().Info(fmt.Sprintf("Pulling %s, size: %s", r.Repo().Reference, utils.ByteFormat(float64(layerSize), 2)))
```

Comment:
Currently, the progress bar provides the user with the name and size of the package they're downloading. IMO since we're eliminating the progress bar, we should upgrade the message with the package name from debug->info

Reply:
Def - thanks for digging into this. Thoughts on a follow up issue?

Reply:
Should be already solved with this. Took the debug message and moved it to info. Diff isn't super clear since I moved the message so it would have size info.